internal/osbuild/result.go

```diff
@@ -6,33 +6,74 @@ import (
 	"io"
 )
 
-type assembler struct {
+type rawAssemblerResult struct {
 	Name    string          `json:"name"`
 	Options json.RawMessage `json:"options"`
 	Success bool            `json:"success"`
 	Output  string          `json:"output"`
 }
 
-type stage struct {
-	Name    string          `json:"name"`
-	Options json.RawMessage `json:"options"`
-	Success bool            `json:"success"`
-	Output  string          `json:"output"`
+type StageResult struct {
```

Comment:
Why did you make this public? It doesn't seem to be used anywhere outside this package.

Reply:
We need to translate this to the koji metadata format, not sure where the translation will live, but i assumed not in this package.